src/style.css

```diff
@@ -1,3 +1,58 @@
+:root {
+  --color-primary: #00acdc; /* Define a variable for primary color */
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.is-active {
+  color: var(--color-primary);
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  background-color: var(--color-primary);
+  border-radius: 8px;
+  height: 4px;
+  width: 100%;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  gap: 20px;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). You should style elements using classes instead of tag names for better maintainability and to avoid conflicts.